DT.Samples.Agora.Shared/AgoraTestConstants.cs

```diff
@@ -15,16 +15,14 @@ public static string AgoraAPI
             }
         }
 
-        /// <summary>
-        /// Temp token generated in https://dashboard.agora.io/ or Token from your API
```

Comment:
this project used with RTC, and we need to be possible use string token OR token server